plane/source/docs/guide-tailsitter.rst

```diff
@@ -0,0 +1,152 @@
+
+All tails-sitters are considered types of QuadPlanes in ArduPilot. You
+should start off by reading the :ref:`QuadPlane documentation
+<quadplane-support` before moving onto this tailsitter specific
+documentation.
+
+Vectored and non-Vectored
+=========================
+
+ArduPilot sub-divides tail-sitters into two broad categories:
+
+- vectored tailsitters can tilt their rotors independently of the
+  movement of the fuselage, giving them vectored thrust
+- non-vectored tailsitters have fixed rotor orientation relative to
+  the fuselage, and rely on large control surfaces for hover authority
+
+Tailsitter Config
+=================
+
+The key parameter to make a plane a tailsitter is to set
+Q_FRAME_CLASS=10. That tells the QuadPlane code to use the tailsitter
+VTOL backend.
+
+The tailsitter backend is a bit unusual, as it is the only
+Q_FRAME_CLASS setting that doesn't have any motors associated with
+it. The way the backend works is that it provides roll, pitch, yaw and
+thrust values to the fixed wing control code. These values then
+control your ailerons, elevons, elevators, rudder and motors.
+
+This has a nice benefit when setting up the tailsitter that you can
+follow the normal fixed wing setup guide in MANUAL and FBWA modes, and
+then when you switch to hover all of your control directions will be
+correct.
+
+It also means that you can fly any fixed wing aircraft that is capable
+of 3D flight as a tailsitter, and fly it in modes like QSTABILIZE,
+QHOVER and QLOITER.
+
+The key differences between fixed wing flight and hovering for a
+tailsitter are:
+
+- when hovering the copter PID gains will be used (the ones starting
+  with Q_A_RAT_*)
+- when in fixed wing flight the fixed wing PID gains will be used (the
+  PTCH2SRV_* and RLL2SRV_* gains)
+- when hovering the nose of the aircraft will try to point up for
+  "level" flight
+- when in fixed wing flight nose of the aircraft will try to point
+  forward for "level" flight
+
+Orientation
+===========
+
+The AHRS_ORIENTATION, the accelerometer calibration and AHRS trim
+should all be done for fixed wing flight. Fixed wing flight is
+considered "normal" orientation for a tailsitter.
+
+Vectored Thrust
+===============
+
+If your tailsitter has vectored thrust then you should set the
+SERVOn_FUNCTION values for your two tilt servos for the left and right
+tilt servos.
+
+For example, if your left tilt servo is channel 5 and your right tilt
+servo is channel 6, then set:
+
+- SERVO5_FUNCTION=75
+- SERVO5_FUNCTION=76
+
+you also need to set the right SERVOn_REVERSED values, and the right
+SERVOn_TRIM, SERVOn_MIN and SERVOn_MAX values.
+
+Vectored Gains
+==============
+
+There are two vectoring gains available. One controls the amount of
+vectored thrust movement in hover, and the other controls the amount
+of vectored thrust movement in forward flight.
+
+The Q_TAILSIT_VHGAIN parameter controls vectored thrust in hover. A
```

Comment:
should link to http://ardupilot.org/plane/docs/parameters.html#q-tailsit-vhgain-tailsitter-vector-thrust-gain-in-hover